src/Distributions.jl

```diff
@@ -18,7 +18,7 @@ import StatsBase: kurtosis, skewness, entropy, mode, modes,
                   fit, kldivergence, loglikelihood, dof, span,
                   params, params!
 
-import PDMats: dim, PDMat, invquad
+import PDMats: dim, PDMat, invquad, ScalMat
```

Comment:
```suggestion
import PDMats: dim, PDMat, invquad
```
don't need ScalMat anymore